examples/aws_soc2.yaml

```diff
@@ -0,0 +1,25 @@
+# A minimal soc2 example.
+#
+# Runs a task that simply lists the default conda environments.
+#
+# Usage:
+#   sky launch -c min-soc2 aws_soc2.yaml
+#   sky down min
+
+name: minimal
+
+resources:
+  cloud: aws
+  # Disk volumes typically need to be encrypted for SOC2 compliance.
+  disk_encrypted: true
```

Comment:
It might be better to keep this field in `~/.sky/config.yaml` instead, as it seems can be applied to all the clusters created on AWS and it supports AWS only for now.

We can add something like the following in `~/.sky/config.yaml`:
```
aws:
    disk_encrypted: true
```

and we can then pass it to the `aws-ray` like the following:
https://github.com/skypilot-org/skypilot/blob/159f97ae9308a8a04a110591701ddf9fd6911da1/sky/clouds/aws.py#L402-L403

Reply:
Great, i'll make that change!